src/pybind/mgr/dashboard/module.py

```diff
@@ -406,6 +411,40 @@ def set_mgr_created_self_signed_cert(self):
             return result
         return 0, 'Self-signed certificate created', ''
 
+    @CLICommand("dashboard get issue")
+    def get_issues_cli(self, issue_number: str):
+        if issue_number == 'None':
+            return -errno.EINVAL, '', 'Invalid arguments. Issue number is missing'
+        cephTrackerClient = CephTrackerClient()
+        try:
+            response = cephTrackerClient.get_issues(issue_number)
+        except RequestException as error:
+            if error.status_code == 404:
+                return errno.EINVAL, '', 'Issue not found'
+            else:
+                return errno.EINVAL, '', 'Unknown error occurred: ' + str(error)
+        return 0, str(response), ''
+
+    @CLICommand("dashboard create issue")
+    def report_issues_cli(self, project: str, tracker: str, subject: str, description: str):
+        '''
+        Create an issue in the Ceph Issue tracker
+        Syntax: ceph dashboard create issue <project> <bug|feature> <subject> <description>
+        '''
+        try:
+            feedback = Feedback(Feedback.Project[project].value,
+                                Feedback.Tracker[tracker].value, subject, description)
+        except KeyError:
+            return -errno.EINVAL, '', 'Invalid arguments'
+        cephTrackerClient = CephTrackerClient()
+        try:
+            response = cephTrackerClient.create_issue(feedback)
+        except KeyError:
+            return -errno.EKEYREJECTED, '', 'Ceph Tracker API key not set'
+        except HTTPError as error:
+            return -errno.EINVAL, '', 'Error while creating request on Ceph Tracker: ' + str(error)
```

Comment:
I didn't have my API key setup and I got this error instead of ` return -errno.EKEYREJECTED, '', 'Ceph Tracker API key not set'`:
```
Error EINVAL: invalid command
[root@ceph ceph]# ceph dashboard create issue dashboard bug test test
2021-08-23T07:01:34.117+0000 7ff07c429700 -1 WARNING: all dangerous and experimental features are enabled.
2021-08-23T07:01:34.126+0000 7ff07c429700 -1 WARNING: all dangerous and experimental features are enabled.
<class 'requests.exceptions.HTTPError'>
```

Reply:
Furthermore, with the endpoint I get a `500` error.

Reply:
fixed